### ISSUE TITLE
Release 4.30

### ISSUE THREAD

**Issue by MohananRahul** [eclipse-platform/eclipse.platform.releng.aggregator#1548]
Umbrella bug to track release activities for 4.30
- [ ] New & Noteworthy
- [ ] Readme file for 4.28 
- [ ] Acknowledgment for 4.28 
- [ ] Tips & Tricks 
- [ ] Migration Guide
- [ ] SWT Javadoc bash for 4.28
- [ ] Publish to Maven central
- [ ] Tag Eclipse 4.28 Release 
- [ ] Contribute to SimRel 
- [ ] Clean up intermediate artifacts (milestones, I-builds and old releases)

Notification : @MohananRahul @deepika-u @alshamams @elsazac @subyssurendran666 @noopur2507 @mpalat @jarthana

**Comment by iloveeclipse** [comment]
@MohananRahul : for RC2 build & release we also need update ecj version to be used to build SDK, as we had ecj change in RC1.

**Comment by MohananRahul** [comment]
> @MohananRahul : for RC2 build & release we also need update ecj version to be used to build SDK, as we had ecj change in RC1.

Yes , We plan to execute it on the release day, RC2 and GA. Can you run https://ci.eclipse.org/jdt/job/copyAndDeployJDTCompiler/ Job (already configured), I dnt have access to run.

**Comment by iloveeclipse** [comment]
Ecj 3.36.0.v20231114-0937 is deployed via https://ci.eclipse.org/jdt/job/copyAndDeployJDTCompiler/158/ to https://repo.eclipse.org/content/repositories/eclipse-staging/org/eclipse/jdt/ecj/3.36.0.v20231114-0937/

**Comment by noopur2507** [comment]
Reviewed N&N docs. No new T&T content is to be added for 4.30.

**Comment by deepika-u** [comment]
SWT Javadoc bash for 4.30
Steps to perform 
- Need to pull the latest changes on master.
- Make sure the git staging view is clean.
- Then run this standalone Java application: JavadocBasher.java.
- If there any valid changes it will reflect in git staging.
- Validate them manually and merge if any.

No changes required to merge.

**Comment by MohananRahul** [comment]
Contribute to SimRel : Done
